lib/Subscription/Blacklist.php

```diff
@@ -0,0 +1,25 @@
+<?php
+namespace MailPoet\Subscription;
+
+class Blacklist {
+  const SALT = 'mailpoet';
+
+  private $blacklist = [
+    'e60c6e0e73997c92d4ceac78a6b6cbbe6249244c4106a3c31de421fc50370ecd' => 1,
+  ];
+
+  public function __construct(array $blacklist = null) {
+    if ($blacklist) {
+      $this->blacklist = array_flip(array_map([$this, 'hash'], $blacklist));
```

Comment:
It would be better to fill the array with `1` or `true` values. Can be done like this:
```
$this->blacklist = array_fill_keys(array_map([$this, 'hash'], $blacklist), 1);
```

Reply:
Good idea, made a change.